shared/git/row/index.tsx

```diff
@@ -1,37 +1,36 @@
-// @flow
 import * as React from 'react'
 import * as Kb from '../../common-adapters'
 import * as Styles from '../../styles'
 
 type Props = {
-  canDelete: boolean,
-  canEdit: boolean,
-  channelName: ?string,
-  chatDisabled: boolean,
-  devicename: string,
-  expanded: boolean,
-  lastEditTime: string,
-  lastEditUser: string,
-  lastEditUserFollowing: boolean,
-  name: string,
-  you: ?string,
-  teamname: ?string,
-  url: string,
-  isNew: boolean,
-  onBrowseGitRepo: () => void,
-  onCopy: () => void,
-  onClickDevice: () => void,
-  onShowDelete: () => void,
-  onChannelClick: (SyntheticEvent<>) => void,
-  onToggleChatEnabled: () => void,
-  onToggleExpand: () => void,
-  openUserTracker: (username: string) => void,
-  _onOpenChannelSelection: () => void,
+  canDelete: boolean
+  canEdit: boolean
+  channelName: string | null
```

Comment:
General comment: why not switch to `channelName?: string`?

Reply:
It wouldn't be a strictly correct conversion, so if the converter did that we'd likely have more tsc errors to deal with. Maybe in TS that'll end up being easier to use, not sure